scripts/watermarkstat

```diff
@@ -254,6 +264,8 @@ def main():
 Examples:
   watermarkstat -t pg_headroom
   watermarkstat -t pg_shared
+  watermarkstat -t q_shared_all
+  watermarkstat -p -t q_shared_all
```

Comment:
For platforms that do not have this queue type, it will error out. Please handle that case

Reply:
If platforms not have this queue type, command print only port name, queue table will be empty.
> $ show queue watermark all
> Egress shared pool occupancy per all queues:
> Port
> \-----------
> Ethernet0
> Ethernet4
> Ethernet8
> Ethernet12
> Ethernet16
> Ethernet20
> Ethernet24
> Ethernet28
> ......